### DISCUSSION TITLE
sidebar/sanfene/os

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#252] [Announcements]
# sidebar/sanfene/os

下载次数超 1 万次，1 万字 51 张手绘图，详解 32 道操作系统面试高频题（让天下没有难背的八股），面渣背会这些 OS 八股文，这次吊打面试官，我觉得稳了（手动 dog）。

https://javabetter.cn/sidebar/sanfene/os.html

**Comment by Kddk-K** [comment]
9.13一刷

**Comment by DrakeWei** [comment]
先来先服务不会导致饥饿吧

**Comment by 9lucifer** [comment]
8.26 1刷

**Comment by garfieldra** [comment]
5.29二刷
